ng2-components/ng2-alfresco-documentlist/src/components/document-list.component.ts

```diff
@@ -67,10 +83,15 @@ export class DocumentListComponent implements OnInit, OnChanges, AfterContentIni
     @Input()
     pageSize: number = DocumentListComponent.DEFAULT_PAGE_SIZE;
 
+    skipCount: number = 0;
+
+    pagination: Pagination;
+
     @Input()
     set rowFilter(value: RowFilter) {
-        if (this.data) {
+        if (this.data && value && this.currentFolderId) {
```

Comment:
Checking for `value` might mean you can never reset row filter by providing `null`.